indexer/CMakeLists.txt

```diff
@@ -149,8 +149,6 @@ file(COPY ${OTHER_FILES} DESTINATION ${CMAKE_BINARY_DIR})
 
 omim_add_library(${PROJECT_NAME} ${SRC})
 
-add_subdirectory(${OMIM_ROOT}/3party/protobuf ${CMAKE_CURRENT_BINARY_DIR}/protobuf)
```

Comment:
Isn't the second param necessary?

Reply:
> Isn't the second param necessary?

Necessary for what?
For the build to work? Apparently not.
For some so far (for me) unclear purpose, that you might know about?
Let me know and I put it back immediately.

Reply:
Is it related to generated protoc binary that can be used to recompile changed map styles?

Reply:
> generated protoc binary 

The `protoc` binary does not appear to be generated by the current `cmake` + `ninja` build process, on the `master` branch:
````bash
find -iname \*proto\*
./indexer/protobuf
./indexer/protobuf/CMakeFiles/protobuf.dir
./OMaps.app/Contents/Resources/drules_proto_dark.bin
./OMaps.app/Contents/Resources/drules_proto_vehicle_clear.bin
./OMaps.app/Contents/Resources/drules_proto_clear.bin
./OMaps.app/Contents/Resources/drules_proto_vehicle_dark.bin
./libprotobuf.a
./drules_struct.proto
````
Have I missed something perhaps?



> recompile changed map styles

From the `STYLES.md` I don't see how and where that is needed, from that it appears that only the already generated python output is used. Once protobuf is istalled with `pip install protobuf`.

The one place where I have found usage of `protoc` is the `generate_proto.sh`, but that just assumes, that the `protoc` is somewhere in the `PATH`.

Reply:
As far as I remember, protoc shoud be the same version to generate styles properly. @vng @pastk do you install some custom protoc/protobuf version or the latest one works fine with styles generation?

Reply:
> As far as I remember, protoc shoud be the same version to generate styles properly. @vng @pastk do you install some custom protoc/protobuf version or the latest one works fine with styles generation?

I don't seem to have a `protoc` binary in my system even. For the styles compiler to work I just installed python protobuf as per the STYLES doc. ATM I'm using `3.19.4`.

Reply:
How about you @vng ?

Reply:
I work on MacOS. brew install protobuf is enough for me.